source/fundamentals/connection/tls.txt

```diff
@@ -29,6 +29,16 @@ You can enable TLS for the connection to your MongoDB instance
 in two different ways: using a property on a ``MongoClientSettings`` object or
 through a parameter in your connection string.
 
+.. note::
+
+   If you connect by using the DNS seedlist protocol, note that the driver enables
+   TLS/SSL by default. To disable it, set the ``tls`` or ``ssl`` parameter value to
+   ``false`` in your connection string or ``MongoClientSettings`` instance.
+
+   To learn more about connection behavior when you use a DNS seedlist,
+   see the :manual:`SRV Connection Format </reference/connection-string/#std-label-connections-dns-seedlist>`
```

Comment:
Based on the text for the link, I think you might be trying to link here instead (the section just below where the link currently takes you)?
```suggestion
   see the :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
```